tutorial/snake/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"image/color"
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/shifter"
+	"tinygo.org/x/drivers/st7735"
+)
+
+const (
+	BUTTON_LEFT = iota
+	BUTTON_UP
+	BUTTON_DOWN
+	BUTTON_RIGHT
+	BUTTON_SELECT
+	BUTTON_START
+	BUTTON_A
+	BUTTON_B
+)
+
+var display st7735.Device
+var buttons shifter.Device
+
+func main() {
+	machine.SPI1.Configure(machine.SPIConfig{
+		SCK:       machine.SPI1_SCK_PIN,
+		MOSI:      machine.SPI1_MOSI_PIN,
+		MISO:      machine.SPI1_MISO_PIN,
+		Frequency: 8000000,
+	})
+	display = st7735.New(machine.SPI1, machine.TFT_RST, machine.TFT_DC, machine.TFT_CS, machine.TFT_LITE)
+	display.Configure(st7735.Config{
+		Rotation: st7735.ROTATION_90,
+	})
+
+	buttons = shifter.NewButtons()
+	buttons.Configure()
+
+	snakeGame := Game{
```

Comment:
Copy improvements from this fix :  https://github.com/tinygo-org/gobadge/commit/0f6354a46bc8154872437c7a7e01bd50b7f97d5b

Reply:
done